app/assets/javascripts/dataTables/locales/zh

```diff
@@ -0,0 +1,31 @@
+/**
+ * Chinese translation
+ *  @name Chinese
+ *  @anchor Chinese
+ *  @author <a href="http://docs.jquery.com/UI">Chi Cheng</a>
+ */
+
+{
+	"sProcessing":   "处理中...",
+	"sLengthMenu":   "显示 _MENU_ 项结果",
+	"sZeroRecords":  "没有匹配结果",
+	"sInfo":         "显示第 _START_ 至 _END_ 项结果，共 _TOTAL_ 项",
+	"sInfoEmpty":    "显示第 0 至 0 项结果，共 0 项",
+	"sInfoFiltered": "(由 _MAX_ 项结果过滤)",
+	"sInfoPostFix":  "",
+	"sSearch":       "搜索:",
+	"sUrl":          "",
+	"sEmptyTable":     "表中数据为空",
+	"sLoadingRecords": "载入中...",
+	"sInfoThousands":  ",",
+	"oPaginate": {
+		"sFirst":    "首页",
+		"sPrevious": "上页",
+		"sNext":     "下页",
+		"sLast":     "末页"
+	},
+	"oAria": {
+		"sSortAscending":  ": 以升序排列此列",
+		"sSortDescending": ": 以降序排列此列"
+	}
+}
```

Comment:
what tool did you use to generate all these? :)

Reply:
Oh I just copy them from here:  https://github.com/DataTables/Plugins/tree/master/i18n

And then use a simple script to rename: https://github.com/code-and-effect/effective_datatables/pull/104/files#diff-44bc3deb2aac18ddfba9896b6f5deaa8

🎉